Automattic/wp-calypso
# Help center: Implement help center entry point

#### Changes proposed in this Pull Request

* This adds the help center entry point to the editor as explained here pbAok1-2D2-p2#entry-point.

#### Testing instructions

1. Run `yarn dev --sync` in `apps/editing-toolkit`.
2. Sandbox a site.
3. Go to the editor of that site.
4. You should see the help button.
5. Click it, it should toggle between active and inactive modes.
